kong/db/strategies/postgres/connector.lua

```diff
@@ -519,10 +519,11 @@ function _mt:query(sql, operation)
   end
 
   local phase = get_phase()
+  local in_admin_api = phase == "content" and ngx.ctx.KONG_PHASE == ADMIN_API_PHASE
 
   if not operation or
-     not self.config_ro or
-     (phase == "content" and ngx.ctx.KONG_PHASE == ADMIN_API_PHASE)
+          not self.config_ro or
+          in_admin_api
```

Comment:
```lua
  if is_admin_api or not (operation or self.config_ro) then
```

Reply:
I've completely reverted the change as it is no longer needed